java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -279,4 +280,38 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Double> zscore(String key, String member);
+
+    /**
+     * Returns the difference between the first sorted set and all the successive sorted sets.
+     *
+     * @see <a href="https://redis.io/commands/zdiff/">redis.io</a> for more details.
+     * @param keys The keys of the sorted sets.
+     * @return An <code>array</code> of elements representing the difference between the sorted sets.
+     *     <br>
+     *     If the first <code>key</code> does not exist, it is treated as an empty sorted set, and the
+     *     command returns an empty <code>array</code>.
+     * @example
+     *     <pre>{@code
+     * String[] payload = client.zdiff(new String[] {"sortedSet1", "sortedSet2", "sortedSet3"}).get();
+     * assert payload.equals(new String[]{"element1"});
```

Comment:
nit
```suggestion
     * assert payload[0].equals("element1");
```

Reply:
A comment with the elements in each of the sorted sets would be nice.  E.g. 
```
sortedSet1 has elements: {"one", "two", "three"}
sortedSet2 has elements: {"two"}
sortedSet3 has elements: {"three"}
```

Reply:
There could have been more elements in the test. Hence I'll keep equals.

Reply:
For the details in examples, we generally don't give background on what has happened before the operation I haven't seen any examples like that in Python and Node. Users could refer to IT tests and Redis documentation.